doc/changes.rst

```diff
@@ -34,13 +34,17 @@ http://varnish-cache.org/docs/trunk/whats-new/index.html and via
 individual releases. These documents are updated as part of the
 release process.
 
-================================
-Varnish Cache 7.6.0 (2024-09-13)
-================================
+===============================
+Varnish Cache NEXT (2025-03-15)
```

Comment:
I don't think we want this commit in a maintenance branch

Reply:
I picked this commit to avoid the conflict in d98d7670956aa3bd3faf31b6d0e56eae0f5ea971, how do we deal with it in this case ? do we add `VDI_EVENT_SICK`  to the previously existing VRT section ?

Reply:
Thanks, I have updated the PR. I removed the commit and fixed the conflict in `vrt.h`. For `changes.rst`, that will be done in a separate PR later this week.